azuredeploy.json

```diff
@@ -85,6 +85,14 @@
           "description": "Id of the keyvault where user credentials for the deployment are stored."
       }
     },
+    "AgentChannel": {
+      "type": "string",
+      "defaultValue": "stable",
+      "allowedValues": ["stable","beta","dev"],
+      "metadata": {
+        "description": "PCoIP Agent channel to use for installer download ."
```

Comment:
small nit - I see there's a space between download and .

Reply:
yes, it is. I will remove the space